services/web/server/src/simcore_service_webserver/projects/_handlers_project_comments.py

```diff
@@ -0,0 +1,294 @@
+""" Handlers for project comments operations
+
+"""
+
+import logging
+from typing import Any
+
+from aiohttp import web
+from models_library.projects import ProjectID
+from models_library.projects_comments import CommentID
+from models_library.rest_pagination import DEFAULT_NUMBER_OF_ITEMS_PER_PAGE, Page
+from models_library.rest_pagination_utils import paginate_data
+from models_library.users import UserID
+from pydantic import BaseModel, Extra, Field, NonNegativeInt
+from servicelib.aiohttp.requests_validation import (
+    parse_request_body_as,
+    parse_request_path_parameters_as,
+    parse_request_query_parameters_as,
+)
+from servicelib.json_serialization import json_dumps
+from servicelib.mimetype_constants import MIMETYPE_APPLICATION_JSON
+from servicelib.rest_constants import RESPONSE_MODEL_POLICY
+
+from .._meta import api_version_prefix as VTAG
+from ..login.decorators import login_required
+from ..security.decorators import permission_required
+from . import projects_api
+from ._handlers_crud import RequestContext
+from .exceptions import ProjectNotFoundError
+
+_logger = logging.getLogger(__name__)
+
+
+routes = web.RouteTableDef()
+
+
+class _CreateProjectCommentsPathParams(BaseModel):
+    project_uuid: ProjectID
+
+    class Config:
+        extra = Extra.forbid
+
+
+class _CreateProjectCommentsBodyParams(BaseModel):
+    content: str
+    user_id: UserID
+
+    class Config:
+        extra = Extra.forbid
+
+
+@routes.post(
+    f"/{VTAG}/projects/{{project_uuid}}/comments", name="create_project_comment"
+)
+@login_required
+@permission_required("project.read")
+async def create_project_comment(request: web.Request):
+    req_ctx = RequestContext.parse_obj(request)
+    path_params = parse_request_path_parameters_as(
+        _CreateProjectCommentsPathParams, request
+    )
+    body_params = await parse_request_body_as(_CreateProjectCommentsBodyParams, request)
+
+    try:
+        # ensure the project exists
+        await projects_api.get_project_for_user(
+            request.app,
+            project_uuid=f"{path_params.project_uuid}",
+            user_id=req_ctx.user_id,
+            include_state=False,
+        )
+
+        if req_ctx.user_id != body_params.user_id:
```

Comment:
As I mention above, I think you should not pass the user_id in the body. 
All comments should automatically belong to the authenticated user!